packages/manager/src/features/CloudPulse/Alerts/AlertResources/AlertsRegionFilter.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import { RegionMultiSelect } from 'src/components/RegionSelect/RegionMultiSelect';
+
+import type { Region } from '@linode/api-v4';
+
+export interface AlertsRegionProps {
+  /**
+   * Callback for publishing the ids of the selected regions
+   */
+  handleSelectionChange: (regions: string[]) => void;
+  /**
+   * The regions to be displayed according to the resources associated with alerts
+   */
+  regionOptions: Region[];
+}
+
+export const AlertsRegionFilter = React.memo((props: AlertsRegionProps) => {
+  const { handleSelectionChange, regionOptions } = props;
+
+  const [selectedRegion, setSelectedRegion] = React.useState<Region[]>([]);
+  return (
+    <RegionMultiSelect
+      onChange={(ids: string[]) => {
+        if (!ids || ids.length === 0) {
```

Comment:
```suggestion
        if (!ids?.length) {
```

zero will also be treated as false